lib/ramble/docs/tutorials/running_an_experiment.rst

```diff
@@ -0,0 +1,270 @@
+.. Copyright 2022-2023 Google LLC
+
+   Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+   https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+   <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+   option. This file may not be copied, modified, or distributed
+   except according to those terms.
+
+.. _running_an_experiment_tutorial:
+
+======================
+Running An Experiment
+======================
+
+This tutorial will provide a basic introduction to navigating Ramble and running
+experiments. In this tutorial, you will set up and run a benchmark simulation
+using `GROMACS <https://www.gromacs.org/>`_, a free and open-source application
+for molecular dynamics.
+
+------------
+Installation
+------------
+
+To install Ramble, see the :doc:`../getting_started` guide.
+
+----------------------
+Available Applications
+----------------------
+
+Once Ramble is installed, viewing the available application definitions is as
+simple as executing:
+
+.. code-block:: console
+
+    $ ramble list
+
+The ``ramble list`` command also takes a query string, which can be used to
+filter available application definitions. For example:
+
+.. code-block:: console
+
+    $ ramble list hp*
+
+might output the following:
+
+.. code-block:: console
+
+    ==> 3 applications
+    hpcc  hpcg  hpl
+
+Additionally, applications can be filtered by their tags, e.g.
+
+.. code-block:: console
+
+    $ ramble list -t molecular-dynamics
+    ==> 4 applications
+    gromacs  hmmer  lammps  namd
+
+The available tags (and their mappings to applications) can be listed using:
+
+.. code-block:: console
+
+    $ ramble attributes --all --tags
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+What's in an application?
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Knowing what applications are available is only part of how a user interacts
+with Ramble. Each application contains one or more workloads, all of which can
+contain their own variables to control their behavior. The ``ramble info``
+command can be used to get more information about a specific application
+definition. As an example:
+
+.. code-block:: console
+
+    $ ramble info gromacs
+
+Will print the workloads and variables the ``GROMACS`` application definition contains.
+
+--------------------
```

Comment:
(I will try to come back to this, after I verify) If we don't use modifiers, we should remove this section from this tutorial.

Reply:
Would it be worth adding a modifier to the tutorial yaml to demo how to use it?

Reply:
I think in a future tutorial. For now, lets keep the intro tutorial pretty simple.

Speaking of which, it might be good to number this tutorial (1) so people know it should go first.